presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -163,6 +163,19 @@ String Functions
 
     Returns the stem of ``word`` in the ``lang`` language.
 
+.. function:: ngrams(string, n) -> array<varchar>
```

Comment:
Please move these up so that the functions within the section are sorted alphabetically.